libs/vkd3d/utils.c

```diff
@@ -28,9 +28,23 @@
 #define DEPTH         (VK_IMAGE_ASPECT_DEPTH_BIT)
 #define STENCIL       (VK_IMAGE_ASPECT_STENCIL_BIT)
 #define DEPTH_STENCIL (VK_IMAGE_ASPECT_DEPTH_BIT | VK_IMAGE_ASPECT_STENCIL_BIT)
+#define PLANAR        (VK_IMAGE_ASPECT_PLANE_0_BIT | VK_IMAGE_ASPECT_PLANE_1_BIT)
 #define TYPELESS      VKD3D_FORMAT_TYPE_TYPELESS
 #define SINT          VKD3D_FORMAT_TYPE_SINT
 #define UINT          VKD3D_FORMAT_TYPE_UINT
+
+static const struct vkd3d_format_footprint nv12_copy_footprints[] =
```

Comment:
iirc, this breaks on some C compilers. At least MSVC tends to complain here.

Reply:
Seems like we've already used this pattern for ages below in the file, so, eh, whatever.

Reply:
what exactly does it complain about?

Reply:
It's the classic problem where a static const variable is not considered a compile time constant, so using a static const variable to feed a new static const variable would break, but it seems to work here, so ... *shrug* I just remember we had issues with that in the shader headers a while back.